packages/main/src/plugin/index.ts

```diff
@@ -738,6 +739,10 @@ export class PluginSystem {
 
     await this.setupSecurityRestrictionsOnLinks(messageBox);
 
+    container.bind<ExperimentalFeatureFeedbackForm>(ExperimentalFeatureFeedbackForm).toSelf().inSingletonScope();
+    const experimentalFeatureFeedbackForm = new ExperimentalFeatureFeedbackForm(configurationRegistry, messageBox);
```

Comment:
it should not use new keyword but perform something like:
(not tested)
```suggestion
    const experimentalFeatureFeedbackForm = container.get<ExperimentalFeatureFeedbackForm>(ExperimentalFeatureFeedbackForm);
```